src/engraving/libmscore/score.h

```diff
@@ -428,7 +428,7 @@ class Score : public QObject, public ScoreElement
     friend class mu::engraving::compat::ReadScoreHook;
     friend class mu::engraving::compat::Read302;
     friend class mu::engraving::Layout;
-    friend class mu::engraving::LayoutContext;
+    friend struct mu::engraving::LayoutContext;
```

Comment:
It seems that both LayoutContext and LayoutMeasure can be removed completely from this file (also the forward declarations at the top).

Reply:
the forware declaration of `class EngravingProject` too...